### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by jweakley**

### CODE REVIEW

**Comment by jweakley** [comment]
This will require a version bump before this is merged.

File: lib/api_me.rb [hunk 1]
```diff
@@ -18,11 +18,11 @@ module ApiMe
   end
 
   module ClassMethods
-    def model(klass) # rubocop:disable TrivialAccessors
+    def model(klass)
       @model_klass = klass
     end
 
-    def serializer(klass) # rubocop:disable TrivialAccessors
+    def serializer(klass)
```
Comment (Samsinite) [inline lib/api_me.rb]: These must no longer be lint errors?

File: lib/api_me/sorting.rb [hunk 2]
```diff
@@ -15,35 +15,34 @@ def results
...
     def sort_meta
-      return Hash.new unless sorting?
+      return {} unless sorting?
       {
-        criteria: sort_criteria.is_blank? || sort_criteria === "" ? default_sort_criteria : sort_criteria,
+        criteria: sort_criteria.is_blank? || sort_criteria === '' ? default_sort_criteria : sort_criteria,
         reverse: sort_reverse,
         record_count: scope.size,
-        total_records: scope.total_count,
+        total_records: scope.total_count
```
Comment (Samsinite) [inline lib/api_me/sorting.rb]: How did this not error before? Or was this in that other PR?

File: lib/api_me/sorting.rb [hunk 3]
```diff
@@ -15,35 +15,34 @@ def results
...
-      if sort_reverse === "true"
-        self.scope = scope.order(criteria => :desc)
-      else
-        self.scope = scope.order(criteria => :asc)
-      end
+      self.scope = if sort_reverse === 'true'
+                     scope.order(criteria => :desc)
+                   else
+                     scope.order(criteria => :asc)
+                   end
```
Comment (Samsinite) [inline lib/api_me/sorting.rb]: I prefer ternary here:
`self.scope = sort_reverse === 'true' ? scope.order(criteria => :desc) : scope.order(criteria => :asc)`
but up to you.

Comment (jweakley) [reply lib/api_me/sorting.rb]: This avoids the line length issue.

**Pull request merged**